lib/index.js

```diff
@@ -11,5 +12,6 @@ var main = function (options, callback) {
 
 module.exports = main;
 module.exports.launch = main;
+module.exports.argv = argv;
```

Comment:
This is needed so that ember-cli-sauce can parse the args given to its own command and pass them on to saucie's `launch`. I think previously saucie parsed the command line options automatically somehow which I don't think is desirable as you might end up getting errors like `unknown option -x` when launching saucie via its API (as opposed to the command line).